platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -278,6 +278,26 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
         //#-end-example-code
     }
     
+    func testMGLMapSnapshotter() {
+        //#-example-code
+        let camera = MGLMapCamera()
+        camera.centerCoordinate = CLLocationCoordinate2D(latitude: 37.7184, longitude: -122.4365)
+        camera.pitch = 20
```

Comment:
The preferred way to create a camera is via one of the other initializers, either `MGLMapCamera(lookingAtCenter:fromEyeCoordinate:eyeAltitude:)` or `MGLMapCamera(lookingAtCenter:fromDistance:pitch:heading:)`. Following MapKit’s lead, these initializers encourage developers to make it clear as to whether the camera is based on a focused point on the map or an actor somewhere in space.

(You can specify whatever distance you want, since `zoomLevel` will override it below. #5583 would allow us to eliminate this awkward situation.)

Reply:
Just moved over the existing documentation, but I will adapt.